package.json

```diff
@@ -17,9 +17,9 @@
   "scripts": {
     "clean": "pnpm npkill -d $(pwd)/packages -t dist && pnpm npkill -d $(pwd) -t node_modules",
     "build": "pnpm --recursive --stream build",
-    "dwn-server": "DWN_SERVER_PACKAGE_JSON=node_modules/@web5/dwn-server/package.json node node_modules/@web5/dwn-server/dist/esm/src/main.js || true",
```

Comment:
For my understanding, why is this removed?

Reply:
don't need it ever since the dwn-server bin got exposed by a PR i did

Reply:
That's awesome!